File: src/main/java/entity/Map.java
package entity;

import java.util.ArrayList;

public class Map 
{
	
    int mapId = 0;
    ArrayList<Node> nodes;
       
    int width;
    int height;
    
    public Map() {
    	nodes = new ArrayList<Node>();
    }

	public int getMapId() {
		return mapId;
	}

	public void setMapId(int mapId) {
		this.mapId = mapId;
	}

	public ArrayList<Node> getNodes() {
		return nodes;
	}

	public void setNodes(ArrayList<Node> nodes) {
		this.nodes = nodes;
	}

	public int getWidth() {
		return width;
	}

	public void setWidth(int width) {
		this.width = width;
	}

	public int getHeight() {
		return height;
	}

	public void setHeight(int height) {
		this.height = height;
	}

	
    

    
}

File: JCIC-Visuals/Assets/Scripts/Map.cs
using System;
using System.Collections.Generic;

public class Map
{
	private Node[,] Nodes;

	public int X = 5;
	public int Y = 5;

	public Map()
	{
		Nodes = new Node[10,10];	
	}

	public Map (JSONObject jsonMap)
	{
		Nodes = new Node[10,10];	
		for (int x = 0; x < 10; x++) {
			for (int y = 0; y < 10; y++) {
				Nodes[x,y] = new Node (jsonMap [y][x]);
			}
		}
	}

	public int Width
	{
		get { return Nodes.GetLength (0); }
	}

	public int Height
	{
		get { return Nodes.GetLength (1); }
	}

	public Node this[int x,int y] {
		get {return Nodes [x, y]; }
		set { Nodes [x, y] = value; }
	}


}


